.github/workflows/release.yml

```
@@ -21,6 +25,6 @@ jobs:
         with:
           publish: pnpm release
           version: pnpm bump
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.11)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)